fhirstore/fhirstore.py

```diff
@@ -269,7 +279,7 @@ def validate(self, resource):
             raise Exception(f"missing schema for resource {resource}")
         validate(instance=resource, schema=schema)
 
-    def search(self, resource_type, args):
+    def search(self, resource_type: str, args: ImmutableMultiDict):
```

Comment:
why do we use this datastructure for args ?

Reply:
It's the one that comes from the flask api